src/Symfony/Component/DependencyInjection/Tests/Compiler/ValidateEnvPlaceholdersPassTest.php

```diff
@@ -390,16 +390,16 @@ class EnvConfigurationWithoutRootNode implements ConfigurationInterface
 {
     public function getConfigTreeBuilder()
     {
-        return new TreeBuilder();
+        return new TreeBuilder('env_extension');
```

Comment:
shouldn't this actually be done in a 4.x branch to avoid using a deprecated API there ?

Reply:
You are right, ill prepare the PR

Reply:
I wonder why that doesn't cause any issues when running the tests on 4.x.

Reply:
see https://github.com/symfony/symfony/pull/31933
I will rebase this when the pr has been merged to master